install/gcp/deployment_manager/istio-cluster.jinja.schema

```diff
@@ -43,11 +43,11 @@ properties:
   installIstioRelease:
     type: string
     description: Install Istio Release version.
-    default: 0.3.0
+    default: 0.5.0
     enum:
-      - 0.2.12
```

Comment:
keep that one and skip 0.3 and 0.4 maybe ?
also maybe wait for 0.5.1
(and broken record, those files don't belong here/in master referencing old versions _ I really think we should move them elsewhere)

Reply:
+1 on keeping 0.2.12 ( quarterly / stable ).

-1 on moving, for now it's good to have them all in one place, and ok to point to old versions.